components/rank-icon.tsx

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+
+type Props = {
+  width: number;
+  height: number;
+  src: string;
+  alt: string;
+};
+const RankIcon = ({ height, width, src, alt }: Props) => {
+  return (
+    <>
+      <Image style={{}} src={src} width={width} height={height} alt={alt} loading="lazy" />
```

Comment:
Very nice 👍 

I think it would be great if we could bring the config for raceToFetch and rankLevel into this component (it would be props)

So the src path would be build here. So you don't need to duplicate it everywhere where you use the component. 

Also can we wrap it to the tooltip? 
https://mantine.dev/core/tooltip/ which would say the number? 

Thanks a lot

Reply:
Pushed some changes with your suggestions. Let me know if this is what u mean.